lib/repl.js

```diff
@@ -65,9 +51,15 @@ exports._builtinLibs = ['assert', 'buffer', 'child_process', 'cluster',
   'smalloc'];
 
 
-function REPLServer(prompt, stream, eval_, useGlobal, ignoreUndefined) {
+const BLOCK_SCOPED_ERROR = 'Block-scoped declarations (let, ' +
+    'const, function, class) not yet supported outside strict mode';
+
+
+function REPLServer(
+    prompt, stream, eval_, useGlobal, ignoreUndefined, replMode) {
```

Comment:
Style: first argument goes on the same line, subsequent arguments go on new lines and should line up.